docs/csi-5to6-upgradepath.md

```diff
@@ -0,0 +1,27 @@
+# upgrade path from PSO 5.2.0 to PSO 6
```

Comment:
Do we need specifically 5.2.0?

Reply:
I dont believe we do, I cant think of any reason prior version won't work. But 5.2.0 is the last official release if I remember correctly, it's also the one I've test with.